app/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,38 @@
+'use client'
+
+import React from 'react';
+
+// components
+import Button from '../Button/Button'
+
+// styles
+import './Dropdown.scss'
+
+const Dropdown = () => {
+    const [open, setOpen] = React.useState(false);
+
+    const handleOpen = () => {
+      console.log("hola")
+      setOpen(!open);
+    };
+
+    const myAccount = <div className='accountText'>My Account</div>
+
+    return(
+        <div className='dropdown'>
+            <Button className='dropdownButton' children={myAccount} onClick={handleOpen} logo='/assets/blackmarket-arrow.png' logoClassName='logoButton'></Button>
```

Comment:
When you define the `children` prop in a component, you actually pass it as what is inside of the component. So in this case: 

```
<Button className='dropdownButton' onClick={handleOpen} logo='/assets/blackmarket-arrow.png' logoClassName='logoButton'>
  {myAccount}
</Button>
```